src/test/java/datawave/data/normalizer/NumericRegexNormalizerTest.java

```diff
@@ -0,0 +1,282 @@
+package datawave.data.normalizer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class NumericRegexNormalizerTest {
+    
+    private static final String[] letters = {"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w",
+            "x", "y", "z", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"};
+    private String regex;
+    
```

Comment:
Given the complexity of this code, the best defense is going to be the robustness of the tests.  What I would like to see is showing the original regex matching a set of numbers (and not matching a set of others).  Then show that the normalized version of the regex matches the same set of numbers in their normalized form and does not match the other numbers in their normalized form.